packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -257,28 +272,74 @@ function WorklistDataGrid(props) {
   );
 
   const handleClearRowSelection = () => {
-    ariaLiveMsg.current = '';
+    let message = '';
+    multiRowSelectionWithLastSelected.current = null;
     if (isAnyRowSelected()) {
-      ariaLiveMsg.current = intl.formatMessage({ id: 'Terra.worklist-data-grid.all-rows-unselected' });
+      message = intl.formatMessage({ id: 'Terra.worklist-data-grid.all-rows-unselected' });
       // Esc (while in row selection mode and rows are selected): Clear selection
       if (onClearSelectedRows) {
         onClearSelectedRows();
       }
     } else if (onDisableSelectableRows) {
-      ariaLiveMsg.current = intl.formatMessage({ id: 'Terra.worklist-data-grid.row-selection-mode-disabled' });
+      multiRowSelection.current = null;
+      message = intl.formatMessage({ id: 'Terra.worklist-data-grid.row-selection-mode-disabled' });
       onDisableSelectableRows();
     }
+    setAriaLiveMsg(message);
+  };
+
+  const handleEnableSelectableRows = (newEndRowIndex, selectionData) => {
```

Comment:
I am a little confused by the method name.  Doesn't "hasRowSelection" enable the ability to select multiple rows?